Project.toml

```diff
@@ -0,0 +1,32 @@
+name = "GeneticVariation"
+uuid = "9bc6ac9d-e6b2-5f70-b0a8-242a01662520"
+authors = ["Ben J. Ward <benjward@protonmail.com>"]
+version = "0.4.1"
+
+[deps]
+Automa = "67c07d97-cdcb-5c2c-af73-a7f9c32a568b"
+BGZFStreams = "28d598bf-9b8f-59f1-b38c-5a06b4a0f5e6"
+BioCore = "37cfa864-2cd6-5c12-ad9e-b6597d696c81"
+BioSequences = "7e6ae17a-c86d-528c-b3b9-7f778a29fe59"
+BufferedStreams = "e1450e63-4bb3-523b-b2a4-4ffa8c0fd77d"
+Combinatorics = "861a8166-3701-5b0c-9a16-15d98fcdc6aa"
+IntervalTrees = "524e6230-43b7-53ae-be76-1e9e4d08d11b"
+Twiddle = "7200193e-83a8-5a55-b20d-5d36d44a0795"
+
+[compat]
+Automa = "0.8"
+BGZFStreams = "0.3"
+BioCore = "2.0.5"
```

Comment:
What's going on here? According to SemVer, this should be the same as 2.0, shouldn't it?

Reply:
No, BioCore has some changes (bug fixes?) that we need. The tests won't pass unless we require at least v2.0.5.